src/local/butler/run_server.py

```diff
@@ -200,7 +200,12 @@ def execute(args):
             datastore_emulator_port=constants.DATASTORE_EMULATOR_PORT,
             log_level=args.log_level,
             pubsub_emulator_host=constants.PUBSUB_EMULATOR_HOST,
-            local_gcs_server_host=constants.LOCAL_GCS_SERVER_HOST))
+            local_gcs_server_host=constants.LOCAL_GCS_SERVER_HOST),
+        extra_environments={
+            # PYTHONPATH on CI includes an outdated App Engine SDK, which has
+            # bugs in dev_appserver.py.
+            'PYTHONPATH': '',
```

Comment:
Do we not need to have path set to ROOT_DIR/src atleast ?

Reply:
nope, not necessary here (and this isn't set in dev environment either).